src/components/ui/button/index.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react"
+import { Slot } from "@radix-ui/react-slot"
+import { cva, type VariantProps } from "class-variance-authority"
+import { cn } from "Utils/cn"
+
+const buttonVariants = cva(
+  "inline-flex items-center justify-center rounded font-bold border border-solid transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-slate-950 disabled:pointer-events-none disabled:opacity-50",
+  {
+    variants: {
+      variant: {
+        contained: "text-white",
+        outlined: "bg-transparent text-prominent"
+      },
+      color: {
+        primary: "border-primary hover:border-primary-hover focus:border-primary-hover",
+        secondary: "border-secondary",
+        success: "border-success",
+      },
+      size: {
+        default: "text-base",
+        sm: "text-[0.75rem] px-3",
+        md: "text-[0.875rem]",
+      },
+      fullwidth: {
+        true: "flex w-full",
+      },
+    },
+    compoundVariants: [
+      {
+        variant: "contained", 
+        color: "primary",
+        className: "bg-primary hover:bg-primary-hover focus:bg-primary-hover"
```

Comment:
```suggestion
        className: "bg-primary hover:bg-primary-hover focus:bg-primary-hover",
```

same for couple of more in this file

Reply:
this is because of eslint error

Reply:
fixed